<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-T1H5H3C" failures="1" tests="22" name="Test" time="76.258" errors="0" timestamp="2024-04-16T13:45:47 IST">
  <testcase classname="testCases.OneTest" name="methodOneTest1" time="0.013"/>
  <testcase classname="testCases.TwoTest" name="methodTwoTest1" time="0.014"/>
  <testcase classname="testCases.ThreeTest" name="methodThreeTest1" time="0.013"/>
  <testcase classname="testCases.FourTest" name="methodFourTest1" time="0.011"/>
  <testcase classname="testCases.OneTest" name="methodOneTest2" time="0.002"/>
  <testcase classname="testCases.OneTest" name="methodOneTest3" time="0.002"/>
  <testcase classname="testCases.ThreeTest" name="methodThreeTest2" time="0.007"/>
  <testcase classname="testCases.ThreeTest" name="methodThreeTest3" time="0.001"/>
  <testcase classname="testCases.TwoTest" name="methodTwoTest2" time="0.005"/>
  <testcase classname="testCases.FourTest" name="methodFourTest2" time="0.012"/>
  <testcase classname="testCases.TwoTest" name="methodTwoTest3" time="0.001"/>
  <testcase classname="testCases.FourTest" name="methodFourTest3" time="0.001"/>
  <testcase classname="testCases.SingleMethodTest" name="accessForgotPasswordScreen" time="2.046">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.SingleMethodTest.accessForgotPasswordScreen(SingleMethodTest.java:23)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- accessForgotPasswordScreen -->
  <testcase classname="testCases.SingleMethodTwoTest" name="loginwithValidandInvalidCredentials" time="4.629"/>
  <testcase classname="testCases.LoginTest" name="loginwithValidandInvalidCredentials" time="2.229"/>
  <testcase classname="testCases.SingleMethodTwoTest" name="loginwithValidandInvalidCredentials" time="9.809"/>
  <testcase classname="testCases.LoginTest" name="loginwithValidandInvalidCredentials" time="6.462"/>
  <testcase classname="testCases.LoginTest" name="loginwithValidandInvalidCredentials" time="6.472"/>
  <testcase classname="testCases.SingleMethodTwoTest" name="loginwithValidandInvalidCredentials" time="10.052"/>
  <testcase classname="testCases.LoginTest" name="validateIncorrectUserName" time="1.761"/>
  <testcase classname="testCases.LoginTest" name="validateIncorrectUserName" time="1.996"/>
  <testcase classname="testCases.LoginTest" name="accessForgotPasswordScreen" time="2.064"/>
</testsuite> <!-- Test -->
